src/main/java/com/redislabs/Connector.java

```diff
@@ -137,7 +127,7 @@ public Connector(String name, String xmlDef, int batchSize, int duration, int re
         .setPattern(streamName)
         .setBatchSize(batchSize)
         .setDuration(duration)
-        .setFailurePolicy(FailurePolicy.RETRY)
+        .setFailurePolicy(FailurePolicy.CONTINUE)
```

Comment:
This would be consider a breaking, we should allow to set it with register the connector and the default should be RETRY.

Reply:
Thinking about it now, you should probably change it to ABORT if the retry mechanism is enable no? If the retry mechanism is enable then you will not raise an exception and will just set the error stream, so if there is an exception raised it must be a bug in the code no?

Reply:
Basically, you're right. Those kind of errors are not supposed to happen in prod. However, we probably want to skip over those just to be safe.

Reply:
I can set it to continue if retryInterval < 0, otherwise retry

Reply:
Added an option to use DLQ in case of non connection related errors. Changed mode back to RETRY

Reply:
I just missed where we use it

Reply:
Right, I moved it to separate exceptions and forgot to put it back in. Just did